examples/custom-action/src/tests/index.ts

```diff
@@ -0,0 +1,78 @@
+import { expect } from "chai";
+import { core } from "@mediarithmics/plugins-nodejs-sdk";
+import "mocha";
+import * as request from "supertest";
+import * as sinon from "sinon";
+import { MyCustomActionPlugin } from "../MyPluginImpl";
+
+describe.only("Test Custom Action example", function () {
+  // All the magic is here
+  const plugin = new MyCustomActionPlugin(false);
+  let runner: core.TestingPluginRunner;
+
+  it("Check the behavior of a dummy custom action", function (done) {
+    const rpMockup: sinon.SinonStub = sinon.stub();
+
+    const properties: core.DataListResponse<core.PluginProperty> = {
+      status: "ok",
+      count: 1,
+      data: [
+        {
+          technical_name: "hello_world",
+          value: {
+            value: "Sacre Hubert",
+          },
+          property_type: "STRING",
+          origin: "PLUGIN",
+          writable: true,
+          deletable: false,
+        },
+      ],
+    };
+
+    rpMockup
+      .withArgs(
+        sinon.match.has(
+          "uri",
+          sinon.match(function (value: string) {
+            return (
+              value.match(
+                /\/v1\/scenarios_custom_actions\/(.){1,10}\/properties/
+              ) !== null
+            );
+          })
+        )
+      )
+      .returns(properties);
+
+    runner = new core.TestingPluginRunner(plugin, rpMockup);
+
+    // We init the plugin
+    request(runner.plugin.app)
+      .post("/v1/init")
+      .send({ authentication_token: "Manny", worker_id: "Calavera" })
+      .end((err, res) => {
+        expect(res.status).to.equal(200);
+      });
+
+    const customActionRequest: core.CustomActionRequest = {
+      user_point_id: "26340584-f777-404c-82c5-56220667464b",
+      instance_id: "62",
+    };
+
+    request(runner.plugin.app)
+      .post("/v1/scenario_custom_action")
```

Comment:
What is the rationale behind using `scenarios_custom_actions` to fetch the plugin properties and `scenario_custom_action` to call the plugin?

Sometimes we like `scenarios` and sometimes we prefer `scenario`?

Personally i don't have a preference but I like consistent designs.